src/components/AssignAttributeDialog/AssignAttributeDialog.tsx

```diff
@@ -217,7 +216,7 @@ const AssignAttributeDialog: React.FC<AssignAttributeDialogProps> = ({
         <ConfirmButton
           transitionState={confirmButtonState}
           color="primary"
```

Comment:
Same as with IconButton, shouldn't we default to `primary` in all instances?

Reply:
We should 🤔